src/test/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/payment/FeesAndStatusCheckPreparerTest.java

```diff
@@ -152,79 +150,6 @@ void should_return_not_available_error_for_old_pa_appeals(Event event) {
         assertTrue(callbackResponse.getErrors().contains("You cannot make a payment for this appeal using Payment by Account"));
     }
 
-    @ParameterizedTest
-    @EnumSource(value = Event.class, names = {
-        "PAYMENT_APPEAL"
-    })
-    void should_return_not_available_error_for_old_ea_appeals(Event event) {
-
```

Comment:
JourneyType isn't set to AIP, so it defaults to LR journey.
LR journey has no access to the "PAYMENT_APPEAL" event.

Test no longer relevant.